cmd/bbolt/main.go

```diff
@@ -536,6 +537,8 @@ func formatBytes(b []byte, format string) (string, error) {
 		return bytesToAsciiOrHex(b), nil
 	case "redacted":
 		return fmt.Sprintf("<redacted len:%d>", len(b)), nil
+	case "sha256":
+		return fmt.Sprintf("<redacted sha256:%x>", bytesToSha256Sum(b)), nil
```

Comment:
Can we modify `redacted` to contain both (len and sha1) ? 

There is no backward compatibility problem, as "redacted" was recently added and there was no bbolt release in the meantime.

Reply:
Done